test/integration/find_local_council_test.rb

```diff
@@ -8,6 +8,10 @@ class FindLocalCouncilTest < ActionDispatch::IntegrationTest
 
   context "when visiting the start page" do
     setup do
+      example_generator = GovukSchemas::RandomExample.for_schema("placeholder", schema_type: "frontend")
```

Comment:
That's interesting observation. Special routes don't have a frontend schema:

> Note that this schema has a restricted set of properties at the metadata level, and no details or links hashes at all. Therefore, it is not constructed by combining sub-schemas. In addition, there is no frontend version, because redirects are never seen by the frontend: everything is handled by the router.

(https://github.com/alphagov/govuk-content-schemas/blob/master/formats/special_route/README.md#L22)

I'm wondering if we should change the format, since the page is tagged with a mainstream browse page and parent.